File: Ohtap E Commerce Domain Models/Development/NetCore/Models Objects/Models Objects/Taxes/ITaxData.cs
using Ohtap.ECommerce.Domain.Models.Objects.Datasets;
using Ohtap.ECommerce.Domain.Models.Objects.Regions;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using CoreObject = Ohtap.ECommerce.Domain.Models.Core.Object;

namespace Ohtap.ECommerce.Domain.Models.Objects.Taxes
{
    public interface ITaxData :  CoreObject.IObjectID, CoreObject.IObjectCode, CoreObject.IObjectName, CoreObject.IObjectOrder, CoreObject.IObjectDescription, CoreObject.IObjectState, ITaxValue
    {
        IRegionData Region { get; set; }
        IDataItemList<IDataItemData> Dataset { get; set; }

    }
}

File: Ohtap E Commerce Domain Models/Development/PHP/Objects/Taxes/ITaxData.php
<?php

namespace Ohtap\ECommerce\DomainModels\Objects\Taxs;


use  Ohtap\ECommerce\DomainModels\Core\Object as  CoreObjects ;


interface ITaxData extends ITaxValue,CoreObjects\IObjectID, CoreObjects\IObjectCode,
    CoreObjects\IObjectDescription, CoreObjects\IObjectName, CoreObjects\IObjectOrder, CoreObjects\IObjectState, CoreObjects\IObjectTitle
{

}
